modules/lib/lib-grid/src/test/resources/test/grid-test.js

```diff
@@ -0,0 +1,108 @@
+var gridLib = require('/lib/xp/grid');
+var assert = require('/lib/xp/testing');
+
+exports.testGet_without_key = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    try {
+        memoryGrid.get({});
+    } catch (e) {
+        assert.assertEquals(`Parameter "key" is required`, e);
+    }
+};
+
+exports.testGet = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    assert.assertEquals('value', memoryGrid.get({
+        key: 'key'
+    }));
+};
+
+exports.testDelete_without_key = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    try {
+        memoryGrid.delete({});
+    } catch (e) {
+        assert.assertEquals(`Parameter "key" is required`, e);
+    }
+};
+
+exports.testDelete = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    memoryGrid.delete({
+        key: 'key'
+    });
+};
+
+exports.testSet_without_key = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    try {
+        memoryGrid.set({});
+    } catch (e) {
+        assert.assertEquals(`Parameter "key" is required`, e);
+    }
+};
+
+exports.testSet_without_ttlSeconds = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+
+    memoryGrid.set({
+        key: 'key',
+        value: 'value'
+    });
+};
+
+exports.testSet = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+
+    memoryGrid.set({
+        key: 'key',
+        value: 'value',
+        ttlSeconds: 2 * 60 * 1000
+    });
+};
+
+exports.testModify_without_key = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    try {
+        memoryGrid.modify({});
+    } catch (e) {
+        assert.assertEquals(`Parameter "key" is required`, e);
+    }
+};
+
+exports.testModify_without_func = function () {
+    var memoryGrid = gridLib.getMap('mapId');
+    try {
+        memoryGrid.modify({
+            key: 'key'
+        });
+    } catch (e) {
+        assert.assertEquals(`Parameter "func" is required`, e);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9567561)
```suggestion
        assert.assertEquals('Parameter "func" is required', e);
```